dbusmock/mockobject.py

```diff
@@ -88,6 +88,36 @@ def format_arg(a):
     return s
 
 
+def _wrap_in_dbus_variant(value):
+    dbus_types = [
+        dbus.types.Array,
+        dbus.types.ByteArray,
+        dbus.types.Int16,
+        dbus.types.ObjectPath,
+        dbus.types.Struct,
+        dbus.types.UInt64,
+        dbus.types.Boolean,
+        dbus.types.Dictionary,
+        dbus.types.Int32,
+        dbus.types.Signature,
+        dbus.types.UInt16,
+        dbus.types.UnixFd,
+        dbus.types.Byte,
+        dbus.types.Double,
+        dbus.types.Int64,
+        dbus.types.String,
+        dbus.types.UInt32,
+    ]
+    if isinstance(value, dbus.String):
+        return dbus.String(str(value), variant_level=1)
+    if type(value) in dbus_types:
+        return type(value)(value.conjugate(), variant_level=1)
```

Comment:
This cannot work for at least `Array`, `ByteArray`, `ObjectPath`, `Struct`, `Dictionary`, and `Signature`. Am I missing something here? I tried to use the `UpdateProperties` method with an array property and got the expected exception that `value` does not have `conjugate`.